client/src/components/App.jsx

```diff
@@ -3,7 +3,11 @@ import LandingPage from './Landing/landingPage.jsx'
 import Charity from '../components/Charity/Charity.jsx';
 import AddItem from './Modals/AddItem.jsx'
 import styles from '../styles/App.css';
```

Comment:
Why are there all these commented out import lines. Just want to verify it is all working, then good to go.